packages/gatsby-source-contentful/src/fetch.js

```diff
@@ -110,22 +109,6 @@ ${formatPluginOptionsForCLI(pluginConfig.getOriginalPluginOptions(), errors)}`)
 
   let contentTypeItems = contentTypes.items
 
-  if (process.env.EXPERIMENTAL_CONTENTFUL_SKIP_NORMALIZE_IDS) {
-    reporter.info(
-      `Skipping normalization of \`.id\`, this means \`sys\` objects will not get a \`.contentful_id\``
```

Comment:
Are those `sys` objects accessible via schema? If so - that's breaking change

Reply:
contentful_id will still available, see discussion